### PR TITLE
CLDR 14.0 keyword fixes

### PR DESCRIPTION
**Pull request by macchiati**

### CODE REVIEW

File: unicodetools/org/unicode/propstest/CheckEmojiProps.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package org.unicode.propstest;
```
Comment (macchiati) [inline unicodetools/org/unicode/propstest/CheckEmojiProps.java:1]: Simple check program

File: unicodetools/org/unicode/tools/emoji/CopyImagesToCldr.java [hunk 2]
```diff
@@ -30,7 +30,9 @@
             .build();
 
     public static void main(String[] args) {
-        String targetDir = CLDRPaths.BASE_DIRECTORY + "tools/cldr-apps/WebContent/images/emoji/";
+	System.out.println("Warning: make sure that the images repo is updated to the latest images,\n"
```
Comment (macchiati) [inline unicodetools/org/unicode/tools/emoji/CopyImagesToCldr.java:33]: Fix to point at new location in CLDR

File: unicodetools/org/unicode/tools/emoji/candidateData.txt [hunk 3]
```diff
@@ -21,19 +21,19 @@ Proposal=L2/19-304
 After=🤭
 U+1FAE2
 Name=face with open eyes and hand over mouth
-Keywords=embarrassment | death | oh no | trouble | shook | shaken | surprise | disbelief | awe | aghast | amazement
+Keywords=amazement | awe | disbelief | embarrass | scared | surprise
```
Comment (macchiati) [inline unicodetools/org/unicode/tools/emoji/candidateData.txt:24]: Fixeds keywords as per spreadsheet

File: unicodetools/org/unicode/tools/emoji/candidateData.txt [hunk 4]
```diff
@@ -180,19 +181,18 @@ Proposal=L2/20-219
...
 After=😶
 U+1FAE5
 Name=dotted line face
-Keywords=invisible | out of sight | hidden | disappeared | introvert | depression | unimportant
+Keywords=depressed | disappear | hide | invisible | introvert
 
 Proposal=L2/20-190
 After=🤰
 U+1FAC3
-Name=pregnant man
```
Comment (nedley) [inline unicodetools/org/unicode/tools/emoji/candidateData.txt:194]: Why is this being removed?

File: unicodetools/org/unicode/tools/emoji/candidateData.txt [hunk 5]
```diff
@@ -203,8 +203,7 @@ U+1FAC3 U+1F3FF
 Proposal=L2/20-190
 After=🤰
 U+1FAC4
-Name=pregnant person
```
Comment (nedley) [inline unicodetools/org/unicode/tools/emoji/candidateData.txt:206]: Why is this being removed?

Comment (macchiati) [reply unicodetools/org/unicode/tools/emoji/candidateData.txt:206]: Sorry, bad merge

File: unicodetools/org/unicode/tools/emoji/candidateData.txt [hunk 6]
```diff
@@ -111,19 +112,19 @@ Proposal=L2/19-378
 After=🤭
 U+1FAE3
 Name=face with peeking eye
-Keywords=can’t look away | captivated | staring | peep
+Keywords=captivate | peep | stare
```
Comment (nedley) [inline unicodetools/org/unicode/tools/emoji/candidateData.txt:115]: I think the change in tense for “captivated” is wrong, this is about the captivatee not the captivator.

**Review by nedley — APPROVED** [review]
LGTM

**Pull request merged**